cypress/integration/like.spec.ts

```diff
@@ -30,3 +30,91 @@ it('likes an episode then finds it in the library', () => {
     });
   });
 });
+
+it.skip('can remove the like from a liked episode', () => {
+  cy.createSupabaseClient().then(async (supabase) => {
+    const { data: podcast } = await supabase
+      .from<SBPodcast>('podcasts')
+      .select('id, episodes (title)')
+      .limit(1)
+      .single();
+    expect(podcast).to.be.not.null;
+    if (podcast === null) return;
+    const titles = podcast.episodes.map(({ title }) => title);
+    cy.visitAndWaitForHydration(`/podcasts/${podcast.id}`);
+
+    // Like all the episodes
+    cy.findAllByTestId('episode-card').each((card, index) => {
```

Comment:
suggestion: let's ditch the test IDs
```suggestion
    cy.findAllByRole('article').each((card, index) => {
```